examples/resources/mutation_policy/resource_cluster_scoped_annotation_mutation_policy.tf

```diff
@@ -31,7 +31,7 @@ resource "tanzu-mission-control_mutation_policy" "cluster_annotation_mutation_po
         {
           key      = "component"
           operator = "NotIn"
-          values   = [
```

Comment:
looks like some git rebase issue?

Reply:
Good catch, I will fix this formatting in another PR